pages/api/places/index.js

```diff
@@ -8,4 +8,20 @@ export default async function handler(request, response) {
     const places = await Place.find();
     response.status(200).json(places);
   }
+
+
+  try {
+    if (request.method === "POST") {
+      const places = request.body;
+      await Place.create(places);
+      response.status(201).json({ status: "You're Place was created." });
+    }
+  } catch (error) {
+    console.error(error);
+    response
+      .status(400)
+      .json({ status: `Something went wrong: ${error.message}` });
+  }
+
```

Comment:
```jsx
 if (request.method === "POST") {
  try {
      const places = request.body;
      await Place.create(places);
      response.status(201).json({ status: "You're Place was created." });
    }
  } catch (error) {
    console.error(error);
    response
      .status(400)
      .json({ status: `Something went wrong: ${error.message}` });
  }
```

I would put the try...catch inside of the if statement as then first it is checked if the method is `POST` and then `try...catch` runs.